Index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Prospex</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+        <link rel="stylesheet" href="/css/style.css">
+</head>
+
+  <body>
+
+  <header class="container-fluid">
+      <h1>Prospex</h1>
+      <nav>
+        <ul>
+          <li><a href="programmes.html" title="Our Programmes">Programmes</a></li>
+          <li><a href="donate.html" title="Donate">Donate</a></li>
+          <li><a href="blog.html" title="Read Our Blog">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+<main class="container-fluid">
+  <section id="mission-statement">
+    <h1>We are Prospex</h1>
+    <p>In Islington, almost half of all children live in poverty. Will you help us change this?</p>
+
+  </section>
+
+  <section if="who-we-are">
+  </section>
+
+  <section id="programmes">
+    <h1>Our Programmes</h1>
+    <ul>
+      <li>
+        <a href="./programmes.html" title="Prospect Street Teams">
+        <h2>Street Teams</h2>
+        </a>
+      </li>
+      <li>
+        <a href="./programmes.html" title="Prospect Skills Workshop">
+        <h2>Skills Workshop</h2>
+        </a>
+      </li>
+      <li>
+        <a href="./programmes.html" title="Prospect Summer Programme">
+        <h2>Street Teams</h2>
+        </a>
+      </li>
+      <li>
+        <a href="./programmes.html" title="Prospect Youth Club">
+        <h2>Youth Club</h2>
+        </a>
+      </li>
+      <li>
+        <a href="./programmes.html" title="Prospect 121 Programmes">
+        <h2>1-2-1 Programmes</h2>
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <section id="fundraising">
+    <h1>Donate Now</h1>
+    <p>Without your support Prospex wouldn't be able to deliver any of our work.</p>
+    <a class="btn btn-primary" href="#fundraising" title="Donate to Prospex now">Donate Now</a>
+    </section>
+
+</main>
+<footer class="container-fluid">
+
+
+</footer>
+
+<div class="donate-widget">
+  <a href="#fundraising" title="Donate to Prospex now">💛Donate Now</a>
```

Comment:
The heart is going to be butted up against the Donate Now here. Is this intentional?